scripts/custom/clean_quantity.py

```diff
@@ -0,0 +1,15 @@
+from scripts import utils
+import re
+
+
+def clean_quantity(raw_input):
```

Comment:
Does this convert the string to a number?
If not, I think you can simply return `float(raw_input)` and None if the input is not conform.

Maybe we should also log a warning when we remove the value because it doesn't satisfy the regex